### PR TITLE
Added html for section 2

### LINKED ISSUE #3: html section 2
We need to create the layout of the section 2

### PR DESCRIPTION
**Pull request by FlashShabash904**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -30,12 +30,23 @@ <h1 class="download__title">Peaceful shopping mindful money</h1>
      </div>
    </section>
    <!-- Секция 2 -->
-   <section>Секция 2</section>
+   <section class="waranty" id="waranty">
+    <div class="download__wrapper">
```
Comment (nagorapro) [inline index.html]: waranty__wrapper

File: index.html [hunk 2]
```diff
@@ -30,12 +30,23 @@ <h1 class="download__title">Peaceful shopping mindful money</h1>
...
+      <div class="waranty__text-block">
+        <h2 class="waranty__title">1-year warranty boost</h2>
+        <p class="waranty__copy">ZEN cards have a very neat feature that will help you whenever your electronic devices start failing after the standard warranty ends, which is pretty common.</p>
+        <p class="waranty__copy">When you buy any electronics with your ZEN shopping Mastercard, their warranty gets extended by one year. How does it work? It simply does. Automatically.</p>
+      </div>
+      <div class="waranty__image-block">
+        <img src="./assets/images/2.jpg" alt="Basket">
+      </div>
+    </div>
```
Comment (nagorapro) [inline index.html:43]: 2 spaces

**Review by nagorapro — COMMENTED** [review]
Need fix

**Review by ghost — COMMENTED** [review]
Correct "warranty" everywhere

**Review by ghost — APPROVED** [review]
Good

**Review by Maiia2023 — APPROVED** [review]

**Review by nagorapro — APPROVED** [review]
Good

**Pull request merged**